### ISSUE TITLE
Improve notability of Embed placeholder failure messages

### ISSUE THREAD

**Issue by dcalhoun** [WordPress/gutenberg#36377]
## What problem does this address?
When attempting to embed an invalid URL, the error message that is shown is quite subtle. It is fairly small text with similar sizing and color to the rest of the Embed block placeholder content. 

## What is your proposed solution?
For the web editor, we could increase the contrast of the error text (e.g. color, size) and/or reduce the amount of information/noise within the default placeholder UI as proposed in https://github.com/WordPress/gutenberg/issues/35357#issuecomment-948206877. The native mobile editor utilizes a different color and icon to differentiate the failure state from the default placeholder state. 

## Screenshots

| Web Failure | Native Mobile Failure |
| - | - |
| ![Embed failure message is easily missed](https://user-images.githubusercontent.com/438664/141147173-e6dd33f1-71da-44cc-94d5-ee4052c93e14.png) | ![Mobile editor error message is more prominent](https://user-images.githubusercontent.com/438664/141147644-732453a9-ddff-4b20-a646-6fb6119ad9f9.PNG) |

## Related

This issue is a subset of an originally broader discussion in https://github.com/WordPress/gutenberg/issues/35357.

**Comment by javierarce** [comment]
Another problem I've noticed is that the message disappears if the embed is left or right-aligned. 

<img width="343" alt="Screenshot 2021-11-15 at 12 19 14" src="https://user-images.githubusercontent.com/4933/141773238-ce751653-7cec-45cd-824d-724c1155bc0a.png">

**Comment by javierarce** [comment]
The way we inform about problems [in the Gallery block](https://github.com/WordPress/gutenberg/issues/34901#issuecomment-921681443) is through error notices, so one possible solution could be to add one on top of the placeholder.

![image](https://user-images.githubusercontent.com/4933/141780568-b4ca54bf-2cd4-4038-b9d0-9e1a746e6f26.png)

[Figma link](https://www.figma.com/file/oTPqvedUxXEknR2jzmLN4E/Improve-noticeability-of-Embed-placeholder-failure-messages?node-id=0%3A1)

**Comment by dcalhoun** [comment]
@javierarce thank you for the further research and proposal! 🙇🏻 The usage of the error notice definitely does improve the notability. The only concern I may have would be the legibility of the [contrast ratio](https://contrastchecker.online/1a7bb7/f4a2a2) of the blue text links on the red background. WDYT?

**Comment by javierarce** [comment]
> The only concern I may have would be the legibility of the contrast ratio of the blue text links on the red background.

Yes, me too… maybe we could use this ticket as an opportunity to improve that contrast for the whole project. 

Another option could be to use the _alert_ color, which has better contrast (although is less noticeable). I think it also makes sense semantically. 

![image](https://user-images.githubusercontent.com/4933/141797662-d1e7b19d-7d32-46bc-b892-6907260ca0e2.png)

**Comment by usyntest** [comment]
Is there a solution to this issue? Is there an officially supported way to customize the styling of the Embed block failure message in the editor (e.g., changing its color) without overriding editor CSS?

**Comment by dcalhoun** [comment]
@usyntest to my knowledge, a solution was never implemented for this. I imagine the two options are contributing a solution to the project or overriding editor styles. The latter could result in unexpected outcomes during future updates.
